src/core/i18n/config.ts

```diff
@@ -0,0 +1,15 @@
+const defaultLocale = "en";
+const timeZone = "Europe/Amsterdam";
+const locales = [defaultLocale, "ru"] as const;
+const localesMap = [
```

Comment:
Is this used somewhere?

Reply:
Yes, this is used in the LocaleSwitcher component in the Components/LocaleSwitcher directory to display all available languages in the select.

Reply:
i'm blind, sorry :D